### PR TITLE
Add app.getspace.Space

### PR DESCRIPTION
**Pull request by friebetill**
Hey, I'm the developer of Space and would like to add Space to Flathub.

Space is a flashcard app that allows one to collaborate on flashcards.

### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub.
- [X] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by friebetill** [comment]
bot, build

File: app.getspace.Space.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{
+    "app-id": "app.getspace.Space",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "space",
+    "separate-locales": false,
+    "finish-args": [
```
Comment (barthalion) [inline app.getspace.Space.json:8]:
```
libGL error: MESA-LOADER: failed to retrieve device information
libGL error: MESA-LOADER: failed to open amdgpu: /usr/lib/x86_64-linux-gnu/GL/default/lib/dri/amdgpu_dri.so: cannot open shared object file: No such file or directory (search paths /usr/lib/x86_64-linux-gnu/GL/default/lib/dri)
libGL error: failed to load driver: amdgpu
libGL error: failed to open /dev/dri/card0: No such file or directory
libGL error: failed to load driver: radeonsi
```

You should add `--device=dri`.

File: app.getspace.Space.desktop [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[Desktop Entry]
```
Comment (barthalion) [inline app.getspace.Space.desktop:1]: This is something that ideally should be part of your tarball, but it's not a blocker.

File: app.getspace.Space.json [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        "--share=network",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--socket=x11"
```
Comment (barthalion) [inline app.getspace.Space.json:12]:
```suggestion
        "--socket=fallback-x11"
```

File: app.getspace.Space.metainfo.xml [hunk 4]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (barthalion) [inline app.getspace.Space.metainfo.xml:1]: This is something that ideally should be part of your tarball, but it's not a blocker.

**Comment by friebetill** [comment]
Thanks @barthalion for your review! I hope I have resolved everything to your satisfaction, if not please reopen the conversations.

**Pull request closed without merging**